dotCMS/src/main/java/com/dotcms/contenttype/business/RelationshipFactoryImpl.java

```diff
@@ -64,7 +64,9 @@ public Relationship byInode(final String inode){
                 throw new NotFoundInDbException("Relationship with inode: " + inode + " not found");
             }
             rel = new DbRelationshipTransformer(results).from();
-        }catch (DotDataException e){
+        } catch (NotFoundInDbException e){
+            Logger.debug(this, e.getMessage());
+        } catch (DotDataException e){
                 Logger.error(this, "Error getting Relationship with inode: " + inode, e);
```

Comment:
I think we need to throw this error and allow the transaction to rollback if required, if we just log and continue we could cause data inconsistencies and also the unit test won't we able to detect errors.